client/src/app/components/managing-body/managing-body.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component, Input, Output, EventEmitter} from '@angular/core';
+
+@Component({
+  selector: 'op-managing-body',
+  templateUrl: 'app/components/managing-body/managing-body.component.html',
+  styleUrls: ['app/components/managing-body/managing-body.component.css'],
```

Comment:
We'll write SASS, please change the file extension.
Also, in PR #35 we'll add `Semantic-UI`, let's wait until we merge that PR and then do a re-roll and write your styles component.